### PR TITLE
GT-2228 Improve styling of accordions

### PR DESCRIPTION
**Pull request by caleballdrin**
## Description
https://jira.cru.org/browse/GT-2228
- Improve styling of accordions [1 Day]
* Change heading font to Source Sans Pro
* Make box more rounded
* Add more spacing between accordions

### CODE REVIEW

File: tool-versions
Comment (wrandall22) [inline tool-versions:1]: This file shouldn't exist.

Comment (dr-bizz) [reply tool-versions:1]: You can remove that file @caleballdrin. I've added it to my PR. Also, it should have been `.tool-versions`

Comment (frett) [reply tool-versions:1]: I think he was referring to the extra `tool-versions` file, and not the `.tool-versions` file. Caleb committed 2 different files.

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+nodejs 18.13.0
```
Comment (dr-bizz) [inline .tool-versions:1]: You can remove this as I've added this. You'll need to merge main into your branch as I made some large changes.

File: src/app/new-page/component/content-accordion/content-accordion.component.css [hunk 2]
```diff
@@ -7,59 +15,57 @@
...
-  opacity: 0;
-}
-
-.accordian.mw568.active .accordion-header i.fa-plus  {
-  opacity: 0;
-}
-.accordian.mw568.active .accordion-header i.fa-minus  {
-  opacity: 1;
+.accordion.mw568.active .accordion-header i.fa-chevron-down {
+  -ms-transform: rotate(180deg);
```
Comment (dr-bizz) [inline src/app/new-page/component/content-accordion/content-accordion.component.css:49]: You no longer need to prefix `transform` as 98% of browsers support it.

https://caniuse.com/?search=transform

File: src/app/new-page/component/content-accordion/content-accordion.component.css [hunk 3]
```diff
@@ -7,59 +15,57 @@
...
+  background-color: var(--kg-gray-medium-dark);
 }
 
 .accordion-header i {
   position: absolute;
   right: 20px;
-  transform: translateY(-50%);
-  transition: all 0.3s ease 0s;
-  top: 50%;
+  -moz-transition: all 0.2s linear;
```
Comment (dr-bizz) [inline src/app/new-page/component/content-accordion/content-accordion.component.css:43]: You don't need to prefixe `transform` - https://caniuse.com/?search=transform`

File: src/app/new-page/component/content-accordion/content-accordion.component.html [hunk 4]
```diff
@@ -1,10 +1,9 @@
 <ng-container *ngIf="ready && sections">
   <ng-container *ngFor="let section of sections; trackBy: trackByFn">
-    <div class="accordian mw568">
+    <div class="accordion mw568">
```
Comment (dr-bizz) [inline src/app/new-page/component/content-accordion/content-accordion.component.html:3]: Thank you for fixing this

File: src/app/new-page/component/content-accordion/content-accordion.component.css [hunk 5]
```diff
@@ -7,59 +15,57 @@
...
+  align-items: center;
 }
 
 .accordion-header:hover {
-  background-color: var(--kg-blue-light);
+  background-color: var(--kg-gray);
+  color: var(--kg-gray-dark);
+}
+.accordion-header:active {
+  background-color: var(--kg-gray-medium-dark);
```
Comment (dr-bizz) [inline src/app/new-page/component/content-accordion/content-accordion.component.css:37]: On mobile, we might need to make the padding less when get below 400px. This isn;t urgent as they could use the App when on smaller devices.

**Review by dr-bizz — CHANGES_REQUESTED** [review]

**Review by dr-bizz — COMMENTED** [review]
Great work!! 🔥

**Comment by dr-bizz** [comment]
Something is wrong with this PR, I doubt you edited 215 files.

**Review by dr-bizz — CHANGES_REQUESTED** [review]
Need to merge the latest version of `master` into your  branch to fix all the conflicts and files that should be already deleted.

File: src/app/page/component/content-accordion/content-accordion.component.css [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  max-height: 400px;
+}
+
+.accordion-content app-content-new-repeater {
+  margin-top: 0;
+}
+
+.accordion-content app-content-new-repeater {
+  margin-top: 0;
+}
```
Comment (frett) [inline src/app/page/component/content-accordion/content-accordion.component.css:70]: you repeated this css rule

**Review by dr-bizz — APPROVED** [review]
The changes look great. Awesome work!

**Pull request merged**